release/readme.in.html

```diff
@@ -37,8 +37,8 @@
             <div class="col-md-8 text-center">
                 <h1><img src="favicon.png" alt="EnergyPlus Logo" width="98" height="98"> EnergyPlus</h1>
                 <p>
-                    Version 22.1.0<br>
-                    31 March 2022<br>
+                    Version @CMAKE_VERSION_MAJOR@.@CMAKE_VERSION_MINOR@.@CMAKE_VERSION_PATCH@<br>
+                    @CMAKE_PROJECTED_RELEASE_DATE@<br>
```

Comment:
Thanks @mjwitte, I added them, and now you can see the diffs.  Honestly most of the diffs this time is about using the CMake variables we already have available to automatically populate the current and previous version numbers.  It will make future updates of the README much more minimal.